dbt/models/sale/schema.yml

```diff
@@ -25,8 +25,12 @@ sources:
             description: '{{ doc("shared_column_sv_is_ptax_outlier") }}'
           - name: sv_is_outlier
             description: '{{ doc("shared_column_sv_is_outlier") }}'
-          - name: sv_outlier_type
-            description: '{{ doc("shared_column_sv_outlier_type") }}'
+          - name: sv_outlier_reason1
+            description: '{{ doc("shared_column_sv_outlier_reason1") }}'
+          - name: sv_outlier_reason1
+            description: '{{ doc("shared_column_sv_outlier_reason2") }}'
+          - name: sv_outlier_reason1
+            description: '{{ doc("shared_column_sv_outlier_reason3") }}'
```

Comment:
**[Nitpick, non-blocking]** Instead of documenting each numbered reason separately, let's just have them share one doc block:

```suggestion
          - name: sv_outlier_reason1
            description: '{{ doc("shared_column_sv_outlier_reason") }}'
          - name: sv_outlier_reason1
            description: '{{ doc("shared_column_sv_outlier_reason") }}'
          - name: sv_outlier_reason1
            description: '{{ doc("shared_column_sv_outlier_reason") }}'
```